packages/shared-components/src/constants.ts

```diff
@@ -149,3 +149,529 @@ export const PRESET_COURSE_COLORS = [
   '#FE7408',
   '#D84B39',
 ]
+
+export const ANIMAL_NAMES = [
```

Comment:
What is the purpose of those? They seem not to be in use at the moment?

Reply:
maybe later :)